Makefile

```diff
@@ -26,10 +26,17 @@ $(notebooks): clean_notebooks lint mypy
 	$(CALLYSTO) $(LOCAL_ROOT_DIR)/$@ > $(LOCAL_ROOT_DIR)/$(@:.py=.ipynb)
 	docker exec -it $(CONTAINER) $(CONTAINER_REPO_DIR)/scripts/publish.sh $(CONTAINER_REPO_DIR)/$@ $(CONTAINER_REPO_DIR)/$(@:.py=.ipynb)
 
+internal_cicd_notebooks:=$(patsubst %.py,%,$(patsubst notebooks/%,internal_cicd_%,$(wildcard notebooks/*.py)))
```

Comment:
what does this do?

Reply:
@xbrianh `patsubst` is a make substitution command.

Specifically, `$(patsubst %.py,%,` does a find and replace of a suffix of `.py` with nothing, and  `$(patsubst notebooks/%,internal_cicd_%,` does a find and replace of a prefix of `notebooks/` with `internal_cicd_`.

This creates the following make targets:

```
make internal_cicd_byod
make internal_cicd_prepare_igv_viewer_input
make internal_cicd_vcf_merge_subsample_tutorial
make internal_cicd_xvcfmerge-array-input
```